README.rst

```diff
@@ -1,12 +1,47 @@
-Orange Network
-==============
+Orange3 Network
+===============
 
-Orange Network is an add-on for Orange_ data mining software package. It
-provides network visualization and analysis tools.
+Orange3-Network is an add-on for Orange_ data mining software package. It
+provides network visualization and network analysis tools.
 
 .. _Orange: http://orange.biolab.si/
 
 Documentation is found at:
 
-http://orange-network.readthedocs.org/
+http://orange3-network.rtfd.io
 
+Installation
+------------
+
+Install from Orange add-on installer through Options - Add-ons.
+
+To install the add-on with pip use
+
+    pip install orange3-network
+
+To install the add-on from source, run
+
+    python setup.py install
+
+To register this add-on with Orange, but keep the code in the development directory (do not copy it to 
+Python's site-packages directory), run
+
+    python setup.py develop
+
+Windows users have to compile the code. Run
+
+    python setup.py build_ext -i --compiler=msvc install
```

Comment:
Yes, you still need a compiler to do it. Either Visual Studio or Mingw32, I can add that to the description, if you want, but this is just an eternal pain with Windows where there are no right answers. Even if I write it, there's a good chance it won't work for some people.